portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,19 +32,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<String> comments = new ArrayList<String>();
-  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    comments.add("What an amazing profile");
-    comments.add("This is very awesome indeed.");
-    comments.add("I want to travel to the Taj Mahal as well");
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
-    String json = convertToJsonUsingGson(comments);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Constant variable

Reply:
I did not understand which one should be the constant variable.

Reply:
private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();

Define this class level constant and reuse in all methods in the class

Reply:
Updated.